samples/subsys/ipc/openamp_rsc_table/remote/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
let's remove the 'remote' dir and move everything up one.

Reply:
done